backend/supervisor/jobs/images.go

```diff
@@ -64,6 +66,8 @@ var OptArgument = map[string]string{
 	"google-on-new-video":                    "on-youtube-video",
 	"google-send-email":                      "send-email",
 	"google-update-signature-email":          "update-signature-email",
+	"linear-create-issue":                    "create-issue",
+	"linear-on-issue-create":                 "",
```

Comment:
no "setup-webhook" param here like in github ?

Reply:
No, with github we have to setup a webhook on each repository we want to listen, with Linear its a global webhook

Reply:
Here I just don't launch any job

Reply:
so why not just remove the line ? To avoid an error while launching the job ?

Reply:
However the line in the above map is mandatory